beast/physicsmodel/stars/ezmist/simpletable.py

```diff
@@ -14,24 +14,27 @@
 
 RuntimeError will be raised when writing to a format associated with missing
 package.
-
-
-.. code-block::python
-
-    >>> t = SimpleTable('path/mytable.csv')
-    # get a subset of columns only
-    >>> s = t.get('M_* logTe logLo U B V I J K')
-    # set some aliases
-    >>> t.set_alias('logT', 'logTe')
-    >>> t.set_alias('logL', 'logLLo')
-    # make a query on one or multiple column
-    >>> q = s.selectWhere('logT logL', '(J > 2) & (10 ** logT > 5000)')
-    # q is also a table object
-    >>> q.plot('logT', 'logL', ',')
-    # makes a simple plot
-    >>> s.write('newtable.fits')
-    # export the initial subtable to a new file
 """
+
```

Comment:
Same. I guess the >>> are telling the test to run it. I suppose the imports are missing or what?